index.html

```diff
@@ -1,78 +1,131 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="author" content="SuperGeeks Team"/>
+    <meta name="description" content="GeekSurvey Project Home Page">
+  </head>
+  <body class="d-flex flex-column min-vh-100">
+    <!-- Navbar starts here -->
+    <!--
+    Elements of the following code are derived from the Start Bootstrap - Business Frontpage v.5.0.7 Template.
+    The template is licensed under MIT and can be found at https://github.com/StartBootstrap/startbootstrap-business-frontpage/blob/master/LICENSE.
+    -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+      <div class="container px-5">
+        <img src="images/gkslogo_dark.png" alt="Logo for the Capstone Team" width="50px" height="50px">
+        <a class="navbar-brand " href="index.html">GeekSurvey</a>
+        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
+          aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav ms-auto mb-2 mb-lg-0">
+            <li class="nav-item"><a class="nav-link" href="index.html">Home</a></li>
+            <li class="nav-item"><a class="nav-link" href="supergeeks.html">Team</a></li>
+            <li class="nav-item"><a class="nav-link" href="process.html">Process</a></li>
+            <li class="nav-item"><a class="nav-link" href="solution.html">Solution</a></li>
+            <li class="nav-item"><a class="nav-link" href="tech.html">Tech</a></li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <!-- Navbar ends here -->
 
-  <link rel="shortcut icon" href="images/supergeeks_logo.png" type="image/x-icon">
+    <!-- Body content starts here -->
+    <div>
+      <!--
+      Elements of the following code are derived from the Start Bootstrap - Business Frontpage v.5.0.7 Template.
+      The template is licensed under MIT and can be found at https://github.com/StartBootstrap/startbootstrap-business-frontpage/blob/master/LICENSE.
+      -->
+      <!-- Business Header -->
+      <header class="bg-dark py-5">
+        <div class="container px-5">
+          <div class="row gx-5 justify-content-center">
+            <div class="col-lg-6">
+              <img class="mx-auto d-block" src="images/geeksurvey_logo_hd.png" width="200px" height="200px">
+              <div class="text-center my-5">
+                <h1 class="display-5 fw-bolder text-white mb-2">The GeekSurvey Project</h1>
+                <p class="lead text-white-50 mb-4">
+                  GeekSurvey is a tool for coordinating research. To read more about GeekSurvey, see the
+                  <a href class="text-white-50"="docs/Gerosa-GeekSurvey.pdf">Capstone project proposal</a> written by the project sponsors.
```

Comment:
should be href="<relative link>"

Reply:
Good catch! Will fix in a quick commit shortly